web-local/config-stashmanagertest.yaml

```diff
@@ -87,13 +87,13 @@ systemOfRecord:
       maxConnectionsPerHost: 5
       latencyAware: true
       partitioner: bop
+      healthCheck:
```

Comment:
These changes and the similar ones elsewhere are unrelated to this PR.  The yaml file was out-of-date and needed to be updated.